packages/react/test/specs/components/Carousel/Carousel-test.tsx

```diff
@@ -150,6 +159,26 @@ describe('Carousel', () => {
 
       expect(pagination.getDOMNode().textContent).toBe(`1 of ${items.length}`)
     })
+
+    it('should not change at arrow left if event is invoked on child element', () => {
```

Comment:
either make starting slide greater than 1 or make it circular, so you make sure that it has the possibility to go to the left.

Reply:
thanks 👍  
added circular prop :)